pkg/workceptor/kubernetes.go

```diff
@@ -229,6 +229,25 @@ func podRunningAndReady() func(event watch.Event) (bool, error) {
 	return inner
 }
 
+func GetTimeoutOpenLogstream(kw *KubeUnit) int {
+	// RECEPTOR_OPEN_LOGSTREAM_TIMEOUT
+	// default: 1
+	openLogStreamTimeout := 1
+	envTimeout := os.Getenv("RECEPTOR_OPEN_LOGSTREAM_TIMEOUT")
+	if envTimeout != "" {
+		var err error
+		openLogStreamTimeout, err = strconv.Atoi(envTimeout)
```

Comment:
Please add a check for 0 and negative. In both cases I think using the default of 1 is a good idea.